CONTRIBUTING.md

```diff
@@ -19,14 +19,13 @@ The [documentation contributor guide][doc-contrib-guide] describes how to contri
 
 If you want to get in touch with us first before contributing, you can use:
 
-* [Strimzi mailing list][mailinglist]
-* [Strimzi Slack workspace][slack]
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```

Comment:
on the sandbox projects page they are using this link, maybe it's somehow direct to the channel? 

https://cloud-native.slack.com/messages/strimzi

Reply:
That links is useless unless you are already registered. So I think we should keep what I have there.